bundles/org.openhab.binding.saicismart/README.md

```diff
@@ -0,0 +1,82 @@
+| tyre-pressure-rear-left    | Number:Pressure          | R          | Pressure rear left                                  | no       |
+| tyre-pressure-rear-right   | Number:Pressure          | R          | Pressure rear right                                 | no       |
+| interior-temperature       | Number:Temperature       | R          | Interior temperature                                | no       |
+| exterior-temperature       | Number:Temperature       | R          | Exterior temperature                                | no       |
+| door-driver                | Contact                  | R          | Driver door open state                              | no       |
+| door-passenger             | Contact                  | R          | Passenger door open state                           | no       |
+| door-rear-left             | Contact                  | R          | Rear left door open state                           | no       |
+| door-rear-right            | Contact                  | R          | Rear right door open state                          | no       |
+| window-driver              | Contact                  | R          | Driver window open state                            | no       |
+| window-passenger           | Contact                  | R          | Passenger window open state                         | no       |
+| window-rear-left           | Contact                  | R          | Rear left window open state                         | no       |
+| window-rear-right          | Contact                  | R          | Rear right window open state                        | no       |
+| window-sun-roof            | Contact                  | R          | Sun roof open state                                 | no       |
+| last-activity              | DateTime                 | R          | Last time the engine was on or the car was charging | no       |
+| last-position-update       | DateTime                 | R          | Last time the Position data was updated             | no       |
+| last-charge-state-update   | DateTime                 | R          | Last time the Charge State data was updated         | no       |
+| remote-ac-status           | Number                   | R          | Status of the A/C                                   | no       |
+| switch-ac                  | Switch                   | R/W        | Control the A/C remotely                            | no       |
+| force-refresh              | Switch                   | R/W        | Force an immediate refresh of the car data          | yes      |
+| last-alarm-message-date    | DateTime                 | R          | Last time an alarm message was sent                 | no       |
+| last-alarm-message-content | String                   | R          | Vehicle message                                     | no       |
+
+
```

Comment:
Examples for .thing and .items are missing.

Reply:
I have added demo.things and demo.items examples.